src/speech.cpp

```diff
@@ -13,18 +13,17 @@ static SpeechBubble nullSpeech = { no_translation( "INVALID SPEECH" ), 0 };
 
 void load_speech( JsonObject &jo )
 {
-    const std::string label = jo.get_string( "speaker" );
     translation sound;
     jo.read( "sound", sound );
     const int volume = jo.get_int( "volume" );
-
-    std::map<std::string, std::vector<SpeechBubble>>::iterator speech_type = speech.find( label );
-    // Construct a vector matching the label if needed.
-    if( speech_type == speech.end() ) {
-        speech_type = speech.emplace( label, std::vector<SpeechBubble>() ).first;
+    for( const std::string &label : jo.get_tags( "speaker" ) ) {
+        std::map<std::string, std::vector<SpeechBubble>>::iterator speech_type = speech.find( label );
+        // Construct a vector matching the label if needed.
```

Comment:
And this map lookup can be simplified via `std::map::operator[]` (it creates the value using the default constructor if the key does not exist within the map.)
```C++
speech[label].emplace_back( SpeechBubble{ sound, volume } );
```
And if you're using `emplace_back` (or `emplace`), that functions already constructs the object for you. You don't give it a full object, only its constructor arguments:
```C++
speech[label].emplace_back( sound, volume );
```